### ISSUE TITLE
Crashes on OCR and will not open again

### ISSUE THREAD

**Issue by josh-morgan117** [OpaitSoftware/TesseractStudio.Net#3]
I was able to OCR one page (Windows 10 64bit), but doing the entire PDF (40MBs) made Tesseract Studio disappear after 20 seconds and will not open again. No error message was given. Have uninstalled, restarted, and reinstalled with a new download, but the program will not open running as admin or with compatibility mode on.

**Comment by OpaitSoftware** [comment]
Please try again with the new version 1.2 and let us know if the problem persists. Thank you.

**Comment by josh-morgan117** [comment]
Nothing changed with the new version, so I went an extra step after uninstalling TessStudio and deleted the Opait folder in C:\Users\myUserName\AppData\Roaming\Opait. I then reinstalled the latest software and it is running again.

The program does still crash when attempting to OCR a 118pg 40MB PDF. I've used gImageReader (another Tesseract third-party program) to OCR the same file and it handled fine, but it doesn't have the greatly overlay like TessStudio does.
